qemu/tests/qemu_img_lock_reject_boot.py

```diff
@@ -27,10 +29,19 @@ def _create_os_snapshot():
         _, snapshot = next(gen)
         QemuImgTest(test, params, env, snapshot).create_snapshot()
 
-    def _verify_write_lock_err_msg(test, output, img_file=None):
+    def _verify_write_lock_err_msg(test, img_file=None):
         test.log.info("Verify qemu-img write lock err msg.",)
         msgs = ['"write" lock',
                 'Is another process using the image']
+        # Check expected error messages directly in the test log
+        log_path = "{0}/latest/test-results/*{1}/debug.log".format(
```

Comment:
You could get the debug.log through `test.logfile` https://github.com/avocado-framework/avocado/blob/eb5418015a304f517822bdeab9570c31919302a0/avocado/core/test.py#L393

Reply:
Let me check this right now. I noticed this in the docs before, but I thought it contains only the actual name of logfile, not the full path. But comment here clearly refers about path. Just a moment :wink:

Reply:
@YongxueHong It works :slightly_smiling_face: Thanks for the catch, it helps to simplify things.

```
JOB LOG    : /root/avocado/job-results/job-2023-03-01T05.06-5788fa2/job.log
 (1/1) Host_RHEL.m9.u0.v0.luks.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.qemu_img_lock.reject_boot_same_img_twice.q35: STARTED
 (1/1) Host_RHEL.m9.u0.v0.luks.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.qemu_img_lock.reject_boot_same_img_twice.q35: PASS (144.82 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```